uai2017/python/assign-senior-program-committee.py

```diff
@@ -68,7 +68,7 @@ def assign_areachair(areachair,paper_number):
 
         if areachair_profile.id not in spc.members:
             print "%s not yet a member of the Senior Program Committee; adding them now" % areachair_profile.id
-            client.add_members_to_group(spc, areachair_profile.id)
+            client.add_members_to_group(spc, str(areachair_profile.id))
```

Comment:
another bug, please do a full test of the assignments and confirm that the add_members_to_group returned a valid response